bench/irmin-pack/trace_stat_summary_utils.ml

```diff
@@ -17,6 +17,100 @@
 type histo = (float * int) list [@@deriving repr]
 type curve = float list [@@deriving repr]
 
+let snap_to_integer ~significant_digits v =
```

Comment:
It's not clear to me what the `~significant_digits` are, can you maybe add a comment for this function?

Reply:
I will, in the next PR.

Reply:
Documenting it made me find bugs.

See https://github.com/mirage/irmin/pull/1416/commits/dcf08cdb433cd95e945ead6c872ab050c8d18685

Reply:
And https://github.com/mirage/irmin/pull/1416/commits/a5416193c88c248eb3f5b55220258710fc809bdc